addons/mass_mailing/static/src/scss/mass_mailing.ui.scss

```diff
@@ -23,7 +23,3 @@ body.editor_enable.o_basic_theme.o_in_iframe {
 .oe_structure {
     width: 100%;
 }
-
-:root {
-    font-size: 14px;
```

Comment:
last sentence of commit message says "This commit removes this style, allowing normal text to be displayed
at 14px."

It should either be without a comma, or "at 16px", otherwise the meaning is wrong.

Reply:
remove all the roots !